frontends/web/src/routes/settings/components/dropdowns/activecurrenciesdropdown.module.css

```diff
@@ -17,17 +17,16 @@
 }
 
 
+:global(.react-select__multi-value):nth-last-child(2) {
+    display: none;
+}
+
 /*
 displays ', ' as a pseudo component for all currency component
 except the last displayed component (the 3rd currency component).
-
-The reason for :nth-last-child(2) instead of just :last-child is because there's an extra
-component created by react-select after the actual last currency component. 
-
-So, we're targetting the second to last component here, 
-which is the last selected currency in the DOM.
 */
-.select:not(.hideMultiSelect) :global(.react-select__multi-value):not(:nth-last-child(2))::after {
+
+.select:not(.hideMultiSelect) :global(.react-select__multi-value):not(:nth-last-child(3))::after {
```

Comment:
Changes in this CSS file are for minor rendering correction of the "," and "..." chars in the active currencies dropdown.

Previous (examples):
AUD,
AUD, CAD, 
AUD, CAD, BRL... (3 selected)
SGD, AUD, CAD... (4 selected)

After:
AUD
AUD, CAD
AUD, CAD, BRL (3 selected)
SGD, AUD, CAD... (4 selected)